src/main/java/org/fundacionjala/coding/rodrigo/HighLow.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.coding.rodrigo;
+
+import java.util.Arrays;
+
+/**
+ * In this little assignment you are given a string of space separated numbers,
+ * and have to return the highest and lowest number.
+ */
+public final class HighLow {
+    /**
+     *
+     */
+    protected HighLow() {
+    }
+
+    /**
+     * Find the high and low.
+     *
+     * @param numbers Input number.
+     * @return the High and Low.
+     */
+    public static String highAndLow(final String numbers) {
+        String low = String.valueOf(Arrays.stream(numbers.split(" "))
+                .mapToInt(Integer::parseInt).min().getAsInt());
+        String high = String.valueOf(Arrays.stream(numbers.split(" "))
+                .mapToInt(Integer::parseInt).max().getAsInt());
+        return high.concat(String.format(" %s", low));
```

Comment:
This may be refactored like this.
```java
        IntSummaryStatistics intSummaryStatistics = Arrays.stream(numbers.split(" "))
                .mapToInt(Integer::parseInt)
                .summaryStatistics();
        return String.format("%d %d", intSummaryStatistics.getMax(), intSummaryStatistics.getMin());
```